examples/refrigerator-app/silabs/data_model/refrigerator-thread-app.matter

```diff
@@ -1641,6 +1644,19 @@ endpoint 0 {
   device type ma_rootdevice = 22, version 1;
 
 
+  server cluster Identify {
```

Comment:
Linter caught this: Identify is not allowed on endpoint 0

Reply:
Oh, I wasn't aware of that. I'll move it to ep1. Thanks